app/src/main/java/com/client/fire_and_water/CheckInFragment.kt

```diff
@@ -0,0 +1,39 @@
+package com.client.fire_and_water
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import com.client.fire_and_water.databinding.FragmentCheckInBinding
+
+class CheckInFragment : Fragment() {
+    private var _binding: FragmentCheckInBinding? = null
+
+    // This property is only valid between onCreateView and
+    // onDestroyView.
+    private val binding get() = _binding!!
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentCheckInBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val network : Network = (activity as MainActivity).network
```

Comment:
How is it used?

Reply:
I expect this to be used when server will work and the fragment will send login and password

Reply:
It's very dangerous to leave code in this form for the future. This is the so-called technical debt.